d3a_api_client/redis_aggregator.py

```diff
@@ -198,6 +198,9 @@ def _on_tick(self, message):
                              function_name="on_tick")
 
     def _on_trade(self, message):
+        if message.get('energy') is None:
+            logging.error(f"Received null energy value {message}")
+            return
```

Comment:
We also need to be prepared for other wrong values. Look at the `price` here. It could also be None.

Reply:
I suggest to create a validation function (or decorator) that checks for the price and energy value. What do you think?

Reply:
I agree , on it